File: tiny-event-sourcing-spring-app/src/main/kotlin/ru/quipy/shop/order/entities/Product.java
package ru.quipy.shop.order.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@AllArgsConstructor
public class Product {
    private final UUID id;
    private final String name;
    @Setter
    private Long price;
}

File: tiny-event-sourcing-spring-app/src/main/kotlin/ru/quipy/shop/order/entities/Product.kt
package ru.quipy.shop.order.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@AllArgsConstructor
data class Product (
    val id: UUID,
    val name: String,
    @Setter
    val price: Long
)